server/chat-plugins/thing-of-the-day.ts

```diff
@@ -156,7 +156,7 @@ class OtdHandler {
 		const nomObj = {
 			nomination: nomination,
 			name: user.name,
-			userids: Object.assign(obj, user.prevNames),
+			userids: [...user.previousIDs],
```

Comment:
iirc this is still slower than `user.previousIDs.slice()` so we might want to just use that.

Reply:
```suggestion
			userids: user.previousIDs.slice(),
```